xlive/dllmain.cpp

```diff
@@ -11,7 +11,7 @@
 #include "H2ConsoleCommands.h"
 #include "H2Config.h"
 #include <sstream>
-#include "Read.h"
+#include "ReadArgumets_INI.h"
```

Comment:
arguments is spelled wrong, also using _ in header files is ugly

ReadIniArguments